.github/CODEOWNERS

```diff
@@ -1,49 +1,63 @@
 # https://help.github.com/articles/about-codeowners/
 
-/distribution/ @a-rodin
+PRIVACY.md @binarylogic @jamtur01
+LICENSE @binarylogic @jamtur01
+CONTRIBUTING.md @binarylogic @jamtur01
+CODE_OF_CONDUCT.md @binarylogic @jamtur01
+NOTICE @binarylogic @jamtur01
+SECURITY.md @binarylogic @jamtur01
+
+.github/workflows./* @jamtur01
```

Comment:
```suggestion
.github/workflows/ @jamtur01
```

Reply:
We probably should move this section to the bottom alongside with `/.github/workflows/e2e.yml`